src/commands/call.ts

```diff
@@ -139,4 +126,49 @@ export class CallCommand extends ApifyCommand<typeof CallCommand> {
             process.exitCode = CommandExitCodes.RunFailed;
         }
     }
+
+    private async resolveActorId(client: ApifyClient, localConfigName: string | undefined, usernameOrId: string) {
+        const { actorId } = this.args;
+
+        // Full ID
+        if (actorId?.includes('/')) {
+            const actor = await client.actor(actorId).get();
+            if (!actor) throw new Error(`Cannot find Actor with ID '${actorId}' in your account.`);
+
+            return {
+                userFriendlyId: actor.username ? `${actor.username}/${actor.name}` : actor.id,
```

Comment:
can this actually happen? i'd guess there will be always a username?

Reply:
I assume there will always be a username for this, good point. Will adjust